lib/Command/V1.pm

```diff
@@ -831,6 +831,11 @@ sub help_options {
             $default_value = "\nDefault value $default_value if not specified";
         }
 
+        my $calculated_default = $property_meta->calculated_default;
+        if (defined $calculated_default) {
```

Comment:
What happens if there is a calculated and regular default? Is that allowed in the initializer?